src/Scenes/Opening.cpp

```diff
@@ -288,6 +323,21 @@ void Opening::draw()
 
     _pathLines->draw(_window);
 
+    //
+    // Draw items
+    //
+    _missionText.setText("Find the magic vagina");
+    std::for_each(  _items.begin(), 
```

Comment:
I wonder if it would be better to do the bounds checking in the `timestep` function? I'm paranoid about doing more in the `draw()` method than drawing.